proto/egm.proto

```diff
@@ -147,7 +147,19 @@ message EgmTestSignals			// Test signals
 
 message EgmMeasuredForce 				// Array of 6 force values for a robot
 {
-    repeated double force = 1;
+    optional bool   fcActive = 1;
+    repeated double force = 2;
```

Comment:
Interesting, they have changed the ID for the `force`, which breaks backward compatibility. But it does not affect us since we don't use this field.

Reply:
Yes, I'm wondering why they couldn't add the `fcActive` field as the second one??? :angry: